src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java

```diff
@@ -69,23 +69,31 @@ public void testTokensNotNull() {
     }
 
     @Test
-    public void testWithMultilineStatements() throws Exception {
+    public void testDeclarationsWithMultilineStatements() throws Exception {
         final String[] expected = {
             "49:21: " + getCheckMessage(MSG_KEY),
             "66:17: " + getCheckMessage(MSG_KEY),
             "74:17: " + getCheckMessage(MSG_KEY),
             "86:10: " + getCheckMessage(MSG_KEY),
             "95:28: " + getCheckMessage(MSG_KEY),
-            "140:39: " + getCheckMessage(MSG_KEY),
-            "173:46: " + getCheckMessage(MSG_KEY),
-            "184:47: " + getCheckMessage(MSG_KEY),
         };
-
         verifyWithInlineConfigParser(
-                getPath("InputOneStatementPerLineMultiline.java"),
+                getPath("InputOneStatementPerLineMultilineOne.java"),
```

Comment:
From the #11163 Description:

> Example of update: https://github.com/checkstyle/checkstyle/commit/8673001d9b0edacfa0d54bec85b26db46d1cce4d
> 
> Test method names and input file name should be similar and describe what we are testing. Example from link above:
> 
> Test method name is testJavadocVariableOnInnerClassFields
> Input file name is InputJavadocVariableOnInnerClassFields.java

Please make the file name as descriptive as the test name.

Reply:
@Vyom-Yadav done